### ISSUE TITLE
onetimeboot option broken in 5.1.0.0

### ISSUE THREAD

**Issue by adamaze** [HewlettPackard/python-redfish-utility#100]
Version: 5.1.0.0
I have gotten a number of things to work using this tool, but i cant getonetimeboot to work.
```
iLORest > bootorder --onetimeboot=Hdd
bootname is : Boot Order Current Settings

bootname is : Boot Order Pending Settings

bootpath is : /redfish/v1/systems/1/bios/boot/settings/

ERROR: list indices must be integers or slices, not str
iLORest >

```
i get this same error on: Gen9,Gen10,Gen11,Alletra 4120

**Comment by adamaze** [comment]
looks like the bug was introduced in the most recent version, as the bootorder command works in 5.0.0.0

**Comment by rajeevkallur** [comment]
Fixed in future 5.2 builds.

**Comment by rajeevkallur** [comment]
Fixed
